compiler/rustc_mir_build/src/build/matches/mod.rs

```diff
@@ -444,7 +444,7 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
             // we lower the guard.
             let target_block = self.cfg.start_new_block();
             let mut schedule_drops = true;
-            // We keep a stack of all of the bindings and type asciptions
+            // We keep a stack of all of the bindings and type descriptions
             // from the parent candidates that we visit, that also need to
```

Comment:
The previous comment was correct. A type ascription is the `: ty` in `expr: ty` or `let pat: ty`.

Reply:
`s/asciptions/ascriptions` not descriptions